### ISSUE TITLE
FSE: Possibly limit blocks available in the header

### ISSUE THREAD

**Issue by apeatling** [Automattic/wp-calypso#35450]
From Josh Ruud:

Never underestimating the ways that users are able to…make things interesting…I’ve been just throwing whatever crazy content I can into my header:

![Screen Shot 2019-08-15 at 5 24 54 PM](https://user-images.githubusercontent.com/1464705/63139748-3b273100-bf94-11e9-9334-287708b8f13b.png)

I wonder if it’s wise to limit the blocks available in the header/footer, as I can see people building out entire pages in the header thinking that they’re just working on one page.

Every page has the same content. Help!

Maybe starting out with items like:

* Logo
* Navigation Menu
* Site Title
* Site Description

Just my $.02

**Comment by apeatling** [comment]
I think we should wait on this one until after launch and get more feedback. If we get many reports that users are adding too much to their headers and getting confused, we can look at limiting blocks.

**Comment by apeatling** [comment]
Adding these to the Not Doing column for now and will move if we hear feedback that this is an issue.

**Comment by eduardozulian** [comment]
I was testing FSE and I think we could keep blocks that give our users some control. While I agree that we could limit to a small number (galleries, for example, don't need to exist on a header), blocks like cover, column, and group could turn a header into something creative.

Here's an example using the group block and a background image:

<img width="1680" alt="Captura de Tela 2019-11-28 às 17 02 33" src="https://user-images.githubusercontent.com/1136160/69829926-ea64dc80-1200-11ea-9752-c135d2172eea.png">

Using the group block for the menu and a cover block with the site title + tagline:

<img width="1679" alt="Captura de Tela 2019-11-28 às 17 08 23" src="https://user-images.githubusercontent.com/1136160/69830101-d5d51400-1201-11ea-8ba8-3e7ac6d03719.png">

**Comment by davipontesblog** [comment]
Closing in favor of the core WordPress FSE implementation in dotCom.
